pallets/rmrk-core/src/lib.rs

```diff
@@ -372,19 +342,12 @@ pub mod pallet {
 			nft_id: T::NftId,
 		) -> DispatchResult {
 			let sender = ensure_signed(origin.clone())?;
-
-			// NFTs::<T>::remove(collection_id, nft_id);
-			// if let Some(kids) = Children::<T>::take(collection_id, nft_id) {
-			// 	for child in kids {
-			// 		Pallet::<T>::burn_nft(origin.clone(), child.0, child.1)?;
-			// 	}
-			// }
-
-			// pallet_uniques::Pallet::<T>::do_burn(collection_id.into(), nft_id.into(), |_, _| {
```

Comment:
Let's keep uniques calls.

Reply:
resolved in https://github.com/rmrk-team/rmrk-substrate/pull/28/commits/8dcfc0d3edcb4673d98c33e0872a8d44930e6ec3